src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.CommonSVTControls.json

```diff
@@ -38,6 +38,23 @@
       ],
       "Enabled": true
     },
+    {
+      "ControlID": "ADO_Repository_AuthZ_Dont_Grant_BuildSvc_Permission_On_Branch",
+      "Description": "Do not grant build service groups excessive permissions on repository branches.",
+      "Id": "Repository120",
+      "ControlSeverity": "Medium",
+      "Automated": "Yes",
+      "MethodName": "CheckBuildServiceAccessOnBranch",
+      "Rationale": "If 'Project Collection Build Service' or 'Project Build Service' groups have excessive permissions on important branches of a repository, then a malicious user can access the repository and tamper its contents, bypassing any defined policies.",
```

Comment:
user can access the repository and tamper its contents, bypassing any defined policies.  --> user can access the repository, tamper its contents and bypass any defined policies ?